bitbucket/resource_repository.go

```diff
@@ -124,6 +132,27 @@ func resourceRepository() *schema.Resource {
 					},
 				},
 			},
+			"parent": {
+				Type: schema.TypeMap,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Optional: true,
+				ForceNew: true,
+				ValidateFunc: func(val interface{}, key string) (warns []string, errs []error) {
+					v := val.(map[string]interface{})
+					if _, ok := v["slug"]; ok == false {
+						errs = append(errs, fmt.Errorf("A repository 'slug' is required when specifying a parent to fork from."))
+					}
+					if _, ok := v["owner"]; ok == false {
+						errs = append(errs, fmt.Errorf("A repository 'owner' is required when specifying a parent to fork from."))
+					}
+					return warns, errs
+				},
+			},
+		},
+		Timeouts: &schema.ResourceTimeout{
```

Comment:
why? default is 10 mins why would it take up to an hour to make an API call?

Reply:
Indeed, this can be stripped out; this is a legacy from a prior implementation, before I realised the cause of the API errors. I did think it was setup for 60s rather than 60 minutes.